src/Microsoft.AspNet.Authentication.OAuth/Notifications/OAuthAuthenticatedContext.cs

```diff
@@ -39,11 +36,6 @@ public class OAuthAuthenticatedContext : BaseContext<OAuthAuthenticationOptions>
         }
 
         /// <summary>
-        /// Gets the JSON-serialized user.
-        /// </summary>
-        public JObject User { get; protected set; }
```

Comment:
Yuck, no. There's a reason the retrieval code wasn't already in the base class. All the providers do it differently, a default client will almost never succeed.